testdata/stackdriver/gateway_access_log.yaml.tmpl

```diff
@@ -1,20 +1,28 @@
 entries:
-- labels:
+- http_request:
+    request_method: "GET"
+    request_url: "http://127.0.0.1:{{ .Vars.ClientPort }}/"
+    server_ip: "127.0.0.1:{{ .Vars.ServerPort }}"
+    protocol: "http"
+    status: 200
+  labels:
     destination_principal: ""
     destination_service_host: server.default.svc.cluster.local
-    protocol: http
```

Comment:
Protocol changed from label to http_request, is that correct?

Reply:
Yeah, I moved it to http request. Since it already exists there and I think it makes sense to just use that field.